Overlapping comparisons always evaluate to true

Got this compiler error on master commit ID `2d07090a6b0bc434548f19cac53167a1647e46cb`:

```
make[2]: *** [CMakeFiles/VVVVVV.dir/build.make:388: CMakeFiles/VVVVVV.dir/src/Textbox.cpp.o] Error 1
/home/tikoldit/git/VVVVVV/desktop_version/src/Entity.cpp:4691:43: warning: overlapping comparisons always evaluate to true [-Wtautological-overlap-compare]
                            if (temp < 30 || temp > -30)
                                ~~~~~~~~~~^~~~~~~~~~~~~
/home/tikoldit/git/VVVVVV/desktop_version/src/Entity.cpp:4688:39: warning: overlapping comparisons always evaluate to true [-Wtautological-overlap-compare]
                        if (temp < 30 || temp > -30)
                            ~~~~~~~~~~^~~~~~~~~~~~~
```

--- a/desktop_version/src/Entity.cpp
+++ b/desktop_version/src/Entity.cpp
@@ -4685,10 +4685,10 @@ void entityclass::entitycollisioncheck()
                     {
                         //ok; only check the actual collision if they're in a close proximity
                         temp = entities[i].yp - entities[j].yp;
-                        if (temp < 30 || temp > -30)
+                        if (temp > -30 && temp < 30)
                         {
                             temp = entities[i].xp - entities[j].xp;
-                            if (temp < 30 || temp > -30)
+                            if (temp > -30 && temp < 30)
                             {
                                 if (entitycollide(i, j)) entities[j].state = entities[j].onentity;
                             }

--- a/mobile_version/src/entityclass.as
+++ b/mobile_version/src/entityclass.as
@@ -3851,9 +3851,9 @@
 							if (entities[j].onentity > 0) {
 								//ok; only check the actual collision if they're in a close proximity
 								temp = entities[i].yp - entities[j].yp;
-								if (temp < 30 || temp > -30) {
+								if (temp > -30 && temp < 30) {
 									temp = entities[i].xp - entities[j].xp;
-									if (temp < 30 || temp > -30) {
+									if (temp > -30 && temp < 30) {
 										if (entitycollide(i, j)) entities[j].state = entities[j].onentity;
 									}
 								}

